lib/job/arGenerateFindingAidJob.class.php

```diff
@@ -74,10 +81,17 @@ public function run($parameters)
         return false;
       }
 
-      // Crank the XML through XSL stylesheet and fix header / fonds URL
-      $eadXslFilePath = $appRoot . '/lib/task/pdf/ead-pdf.xsl';
+      // Use XSL file selected in Finding Aid model setting
+      $findingAidModel = 'bombardier';
```

Comment:
Can we call this something more generic? This is meant to go into mainline AtoM right?